build.gradle

```diff
@@ -17,7 +17,7 @@ dependencies {
 
 test {
     useTestNG() {
-        suites "src/test/resources/Homework19.xml"
+        suites "src/test/resources/Homework19MG.xml"
     }
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221350651-52d4cd00-c1db-4f7e-946c-1e0d83f33aea.png)

![image](https://user-images.githubusercontent.com/104761482/221350689-c64c5c8d-bb09-4a4a-9b4e-730c667cc85f.png)